src/containers/Pomodoro/Pomodoro.js

```diff
@@ -36,17 +42,36 @@ class Pomodoro extends Component {
     const { seconds } = this.state;
     const timeout = setTimeout(() => {
       clearInterval(timer);
-      this.setState({ timerIsRunning: false, seconds: seconds, sessionSoundStatus: Sound.status.PLAYING });
+      if (Notification.permission === 'granted') {
+        console.log('asdf');
+        const notification = new Notification(
+          "Session expired! It's break time!"
+        );
+      } else {
+        console.log('permission denied');
+      }
+      this.setState({
+        timerIsRunning: false,
+        seconds: seconds,
+        sessionSoundStatus: Sound.status.PLAYING
+      });
     }, seconds * 1000);
   };
 
+  stopSessionEndingSound = () => {
```

Comment:
If there ain't gonna be any other sounds I think calling this `stopSound` will suffice 😝

Reply:
There are going to be some other sounds, hence the long name ;)